evalml/tests/component_tests/test_imputer.py

```diff
@@ -26,7 +26,7 @@ def imputer_test_data():
         "int with nan": [np.nan, 1, 0, 0, 1],
         "float with nan": [0.0, 1.0, np.nan, -1.0, 0.],
         "object with nan": ["b", "b", np.nan, "c", np.nan],
-        "bool col with nan": pd.Series([True, np.nan, False, np.nan, True], dtype='boolean'),
+        "bool col with nan": pd.Series([True, np.nan, False, np.nan, True]),
```

Comment:
Should this say `dtype='category'` ?
```python
print(pd.Series([True, np.nan, False, np.nan, True]).dtype)
```
yields object dtype

Reply:
I'll make that change!